### ISSUE TITLE
[OpenMetadata version 1.0.0]Quickstart command metadata docker --help raises error raise source.error('global flags not at the start ' re.error: global flags not at the start of the expression at position 1

### ISSUE THREAD

**Issue by HongQuanVu** [open-metadata/OpenMetadata#11371]
**Affected module**
Quick Start docker
**Describe the bug**

Follow the instruction of Quickstart  for OpenMetadata version 1.0.0 [Latest]to run $metadadata docker --help but hits the below error : 
(env) bash-3.2$ metadata docker --help
Traceback (most recent call last):
  File "/Users/macbook/openmetadata-docker/env/bin/metadata", line 5, in <module>
    from metadata.cmd import metadata
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/cmd.py", line 21, in <module>
    from metadata.cli.backup import UploadDestinationType, run_backup
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/cli/backup.py", line 22, in <module>
    from metadata.cli.utils import get_engine
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/cli/utils.py", line 18, in <module>
    from metadata.generated.schema.entity.services.connections.database.mysqlConnection import (
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/generated/schema/entity/services/connections/database/mysqlConnection.py", line 12, in <module>
    from metadata.ingestion.models.custom_pydantic import CustomSecretStr
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/ingestion/models/custom_pydantic.py", line 22, in <module>
    from metadata.utils.logger import ingestion_logger
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/utils/logger.py", line 20, in <module>
    from metadata.generated.schema.api.lineage.addLineage import AddLineageRequest
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/generated/schema/api/lineage/addLineage.py", line 11, in <module>
    from ...type import basic, entityLineage
  File "/Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages/metadata/generated/schema/type/basic.py", line 96, in <module>
    class EntityLink(BaseModel):
  File "pydantic/main.py", line 178, in pydantic.main.ModelMetaclass.__new__
  File "pydantic/typing.py", line 399, in pydantic.typing.resolve_annotations
    raise TypeError("Cannot subclass special typing classes")
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/typing.py", line 366, in _eval_type
    return t._evaluate(globalns, localns, recursive_guard)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/typing.py", line 864, in _evaluate
    eval(self.__forward_code__, globalns, localns),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 1, in <module>
  File "pydantic/types.py", line 463, in pydantic.types.constr
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/re/__init__.py", line 227, in compile
    return _compile(pattern, flags)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/re/__init__.py", line 294, in _compile
    p = _compiler.compile(pattern, flags)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/re/_compiler.py", line 743, in compile
    p = _parser.parse(p, flags)
        ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/re/_parser.py", line 980, in parse
    p = _parse_sub(source, state, flags & SRE_FLAG_VERBOSE, 0)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/re/_parser.py", line 455, in _parse_sub
    itemsappend(_parse(source, state, verbose, nested + 1,
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/re/_parser.py", line 841, in _parse
    raise source.error('global flags not at the start '
re.error: global flags not at the start of the expression at position 1
(env) bash-3.2$ 

**To Reproduce**

Screenshots or steps to reproduce

  mkdir openmetadata-docker && cd openmetadata-docker
  python3 -m venv env
  source env/bin/activate
  pip3 install --upgrade pip setuptools
  pip3 install --upgrade "openmetadata-ingestion[docker]"
  metadata docker --help

**Expected behavior**
No error raised.
**Version:**
 - OS: AWS Ubuntu 22 
 - Python version: python 3
 - OpenMetadata version: 1.0.0 
 - OpenMetadata Ingestion package version: 1.0.0

(env) bash-3.2$ pip3 show "openmetadata-ingestion"
Name: openmetadata-ingestion
Version: 1.0.0.0
Summary: Ingestion Framework for OpenMetadata
Home-page: https://open-metadata.org/
Author: OpenMetadata Committers
Author-email: 
License: Apache License 2.0
Location: /Users/macbook/openmetadata-docker/env/lib/python3.11/site-packages
Requires: antlr4-python3-runtime, avro, boto3, cached-property, chardet, commonregex, croniter, cryptography, email-validator, google, google-auth, grpcio-tools, idna, importlib-metadata, Jinja2, jsonschema, mypy-extensions, openmetadata-sqllineage, pydantic, pymysql, python-dateutil, python-jose, PyYAML, requests, requests-aws4auth, setuptools, sqlalchemy, tabulate, typing-compat, typing-inspect, wheel
Required-by:

**Additional context**  : 
I follow the instruction to setup : https://docs.open-metadata.org/v1.0.0/quick-start/local-deployment

**Comment by ayush-shah** [comment]
Hello @HongQuanVu - we currently don't support 3.11, we are supporting >=3.7 <=3.10 for now..
looks like the dependencies we have in the project are causing some conflicts, some library issues with 3.11
let me check it once if we can have support for the same with minimum conflicts / workarounds.

**Comment by ayush-shah** [comment]
@HongQuanVu python3.11 will fail on installation.. would recommend trying the same in 3.10

**Comment by ayush-shah** [comment]
Closing it as it is duplicate of: https://github.com/open-metadata/OpenMetadata/issues/8359 / https://github.com/open-metadata/OpenMetadata/issues/5878
